src/Parsing/Impl/Parser.cs

```diff
@@ -2264,7 +2274,7 @@ struct Name {
             var itemWhiteSpace = MakeWhiteSpaceList();
 
             var items = ImmutableArray<WithItem>.Empty
-                .Add(ParseWithItem(itemWhiteSpace));;
+                .Add(ParseWithItem(itemWhiteSpace)); ;
```

Comment:
There's a weird extra `;` at the end of this line. Not sure if there are anymore, but the reformat shows it in the diff so I noticed it.